README.md

```diff
@@ -4,8 +4,12 @@
 
 <br>
 
-[![](https://img.shields.io/badge/made%20by-Elrond%20Network-blue.svg?style=flat-square)](http://elrond.com/)
-[![](https://img.shields.io/badge/project-Elrond%20Network%20Testnet-blue.svg?style=flat-square)](http://elrond.com/)
+[![](https://img.shields.io/badge/made%20by-Elrond%20Network-blue.svg)](http://elrond.com/)
+[![](https://img.shields.io/badge/project-Elrond%20Network%20Testnet-blue.svg)](http://elrond.com/)
```

Comment:
Should this link be to https://testnet.elrond.com?

Reply:
yes! changed